src/protocols/tendermint/contract.js

```diff
@@ -0,0 +1,12 @@
+module.exports = class TendermintContract {
```

Comment:
I think you can delete this file 🙂 

It's only required if you call `validateContractValues` in the `validation` step, but it's empty in `tendermint`'s `subgraph.js` file (`immutable.List()`).